torchmetrics/functional/classification/f_beta.py

```diff
@@ -50,22 +51,32 @@ def _fbeta_compute(
     num = (1 + beta**2) * precision * recall
     denom = beta**2 * precision + recall
     denom[denom == 0.] = 1  # avoid division by 0
+    # if classes matter and a given class is not present in both the preds and the target,
+    # computing the score for this class is meaningless, thus they should be ignored
+    if average == AverageMethod.NONE and mdmc_average is None:
+        # a class is not present if there exists no TPs, no FPs, and no FNs
+        meaningless_indeces = torch.nonzero(np.logical_not(np.logical_or(tp.cpu(),
+                                                                         np.logical_or(fp.cpu(), fn.cpu()))))
```

Comment:
please only use pytorch operations
```suggestion
        meaningless_indeces = ((tp | fn | fp) == 0).nonzero()
```